examples/file-uploader/js/list.js

```diff
@@ -0,0 +1,87 @@
+const Type = require('union-type');
+const T = require('ramda/src/T')
+    , adjust = require('ramda/src/adjust')
+    , append = require('ramda/src/append')
+    , curry  = require('ramda/src/curry')
+;
+const h = require('snabbdom/h');
+
+const upload = require('./upload');
+const uploader = require('./uploader');
+
+const sync = (s) => [s, []];
```

Comment:
I've used such a function myself in an app of mine. I've named it `noEffect`. I think of the returned futures as "effects" because they do side-effects.

I'm not sure which name is best though ;)